backend/presentation/tests.py

```diff
@@ -213,6 +212,138 @@ def test_delete_presentation_as_admin(self):
         self.assertEqual(Presentation.objects.count(), 0)
 
 
+class TestFiltersPresentationViewSet(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.user_1 = User.objects.create_user(
+            username="user_1",
+            password="password123",
+            email="user_1@mail.com",
+            first_name="user",
+            last_name="test_1",
+        )
+        cls.user_2 = User.objects.create_user(
+            username="user_2",
+            password="password123",
+            email="user_2mail.com",
+            first_name="user",
+            last_name="test_2",
+        )
```

Comment:
Let's add [factory_boy](https://factoryboy.readthedocs.io/en/stable/) to make things cleaner.